alphastats/gui/utils/analysis.py

```diff
@@ -449,4 +448,20 @@ def _do_analysis(self):
             covar=self._parameters["covar"],
             between=self._parameters["between"],
         )
-        return ancova_analysis, None, self._parameters
+        return ancova_analysis, None
+
+
+ANALYSIS_OPTIONS = {
```

Comment:
would moving the constants to the beggining of the file make more sense?

Reply:
totally, but this needs to be done after the class definitions referenced in the dict